src/Uno.UI.Runtime.Skia.Wpf/WpfHost.cs

```diff
@@ -82,7 +82,7 @@ protected override Task RunLoop()
 
 	private void InitializeDispatcher()
 	{
-		Windows.UI.Core.CoreDispatcher.DispatchOverride = (d, p) => _dispatcher.BeginInvoke(d, p == Uno.UI.Dispatching.NativeDispatcherPriority.Idle ? DispatcherPriority.SystemIdle : DispatcherPriority.Normal);
+		Windows.UI.Core.CoreDispatcher.DispatchOverride = (d, p) => _dispatcher.BeginInvoke(d, p == Uno.UI.Dispatching.NativeDispatcherPriority.Idle ? DispatcherPriority.SystemIdle : DispatcherPriority.Render);
```

Comment:
Why was this change necessary?

Reply:
It's required for this change:
> [Bugfix] Fix deadlock when animating CompositionObjects' sub-properties

Reply:
note that this issue only affected WPF, GTK wasn't affected